app/locker/data/dashboards/BaseDashboard.php

```diff
@@ -34,17 +37,8 @@ public function getGraphData(\DateTime $startDate = null, \DateTime $endDate = n
    *
    **/
   public function statementCount(){
-
-    $collection = \DB::getCollection('statements');
-
-    $query = [];
-    
-    //Limit by LRS
-    if( $this->has_lrs ){
-      $query['lrs_id'] = new \MongoId($this->lrs);
-    }
-
-    return $collection->count($query);
+    $lrs_id = $this->has_lrs ? $this->lrs : null;
```

Comment:
If there's a method that *needs* to be called externally before this method to set `$this->has_lrs` can that be commented somewhere?